mshadow/base.h

```diff
@@ -285,7 +285,11 @@ const unsigned kRandBufferSize = 1000000;
 /*! \brief pi  */
 const float kPi = 3.1415926f;
 /*! \brief type that will be used for index */
-typedef int64_t index_t;
+#if MSHADOW_INT64_TENSOR_SIZE == 1
```

Comment:
shouldnt this be MSHADOW_TENSOR_SIZE_INT64 since we're saying that setting this to 1 will make tensor sizes use int64 datatypes?